boot/mbed/README.md

```diff
@@ -0,0 +1,41 @@
+# MCUboot port for Mbed OS
```

Comment:
This file has to be moved to `/docs/readme-mbedos.md` and a proper entry added to `/docs/index.md`; has to update the link to `mbed_lib.json` afterwards.